### PR TITLE
RRD Asthma: Tweaks asthma PDF with slight formatting changes

### PR DESCRIPTION
**Pull request by allthesignals**
## Description of change
- medications: include label even if there is no data for route, refills, etc.
- wrap message in table so that it can have background color
- increase leading (space between lines of text)

## Original issue(s)
https://www.notion.so/Generate-asthma-PDF-10f4de575ee3463fa08265761b45bd9a

## Things to know about this PR
Includes some squashed duplicate commits bc this is a branch of a PR'd branch.

### CODE REVIEW

File: app/services/rapid_ready_for_decision/views/shared/medications.erb [hunk 1]
```diff
@@ -35,25 +40,19 @@
         <%= medication['dosageInstructions'].join('; ') %><br />
       <% end %>
 
-      <% if medication['authoredOn'].present? %>
-        Prescribed on: <%= medication['authoredOn'][0, 10].to_date.strftime('%m/%d/%Y') %>
-      <% end %>
+      <%# TODO: Add test for additional fields and make sure to include additional medications %>
```
Comment (allthesignals) [inline app/services/rapid_ready_for_decision/views/shared/medications.erb:43]: Actually, given what we learned about nil values, we should include the "present?" check...

File: app/services/rapid_ready_for_decision/views/shared/medications.erb [hunk 2]
```diff
@@ -10,12 +10,17 @@
...
-  See "How we identify asthma prescriptions" at the end of this document for more details.
+  <p> <br /> </p>
+  <table>
+    <tr><td>
+      <br/>
+      <b>Important!</b> Medications that <b><i>may</i></b> have been prescribed for asthma are marked
+      with <font name="DejaVuSans">&#9654;</font>arrows<font name="DejaVuSans">&#9664;</font>.
+      Claims adjudicators should check the patient's medical records to see what the medication was
+      prescribed for.<br/>
+      See "How we identify asthma prescriptions" at the end of this document for more details
```
Comment (yoomlam) [inline app/services/rapid_ready_for_decision/views/shared/medications.erb:21]:
```suggestion
      See "How we identify asthma prescriptions" at the end of this document for more details.
```

**Review by yoomlam — APPROVED** [review]

File: app/services/rapid_ready_for_decision/views/shared/medications.erb (hunk 2 above)
Comment (allthesignals) [reply app/services/rapid_ready_for_decision/views/shared/medications.erb:21]: Ack, thanks. Merge conflict regression.

**Review by yoomlam — APPROVED** [review]

File: app/services/rapid_ready_for_decision/views/shared/medications.erb [hunk 3]
```diff
@@ -35,25 +40,31 @@
         <%= medication['dosageInstructions'].join('; ') %><br />
       <% end %>
 
+      <%# TODO: Add test for additional fields and make sure to include additional medications %>
+
+      Prescribed on:
```
Comment (LindseySaari) [inline app/services/rapid_ready_for_decision/views/shared/medications.erb:45]: Should the Prescribed on, Duration and Refills be nested within the conditional statement?

Comment (yoomlam) [reply app/services/rapid_ready_for_decision/views/shared/medications.erb:45]: According to the ticket description, no:
> medications: include label even if there is no data for route, refills, etc.

I'll let @allthesignals confirm that interpretation.

Comment (allthesignals) [reply app/services/rapid_ready_for_decision/views/shared/medications.erb:45]: Yep, thanks for raising this! We tried it both ways and designers preferred label + no data in those situations.

Comment (LindseySaari) [reply app/services/rapid_ready_for_decision/views/shared/medications.erb:45]: Very nice! I'll approve once the changes are all set

Comment (allthesignals) [reply app/services/rapid_ready_for_decision/views/shared/medications.erb:45]: @LindseySaari thanks, Lindsey, the changes should be good to go.

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**